crates/services/gas_price_service/src/v1/uninitialized_task.rs

```diff
@@ -100,7 +104,7 @@ where
     L2DataStore: L2Data,
     L2DataStoreView: AtomicView<LatestView = L2DataStore>,
     AtomicStorage: GasPriceServiceAtomicStorage,
-    DA: DaBlockCostsSource,
+    DA: DaBlockCostsSource + 'static,
```

Comment:
do we need static lifetime here?

Reply:
Maybe not, but the `ServiceRunner` expects it, so that's why we require it at this level.